timemachine/fe/single_topology_v3.py

```diff
@@ -177,10 +177,59 @@ def setup_dummy_interactions(
     )
 
 
+def canonicalize_improper_idxs(idxs):
+    """
+    Canonicalize an improper_idx while being symmetry aware.
+
+    Given idxs (i,j,k,l), where i is the center, and (j,k,l) are neighbors:
+
+    0) Canonicalize the (j,k,l) into (jj,kk,ll) by sorting
+    1) Generate clockwise rotations of (j,k,l)
+    2) Generate counter clockwise rotations of (j,k,l)
```

Comment:
```suggestion
    1) Generate clockwise rotations of (jj,kk,ll)
    2) Generate counter clockwise rotations of (jj,kk,ll)
```
(otherwise step (0) appears unreferenced)